src/tables.js

```diff
@@ -9,6 +9,7 @@ const Flow = {
   STEM_HEIGHT: 32,
   STAVE_LINE_THICKNESS: 2,
   RESOLUTION: 16384,
+  INITIAL_ARTICULATION_OFFSET: -0.5,
```

Comment:
Are these values supposed to be configuration? If they are, why scatter the config across the modules instead of a centralized location? If it's not configuration, then I agree.